.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,50 @@
+name: Publish Documentation Site
+
+on:
+  push:
+    branches: [stromberg/manual-publish]
+  workflow_dispatch:
```

Comment:
Haven't seen this one. Could you add a comment documenting why it's needed here?

Reply:
This allows us to manually kick off the workflow. We may want to (eventually?) move this to automatically happen on commits to `main`, but for now it's a manual process that lets you choose the git ref to use for building the docs.